core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.ts

```diff
@@ -170,8 +174,11 @@ export class StoryEditorNavbarComponent implements OnInit {
     modalRef.result.then((commitMessage) => {
       this.storyEditorStateService.saveStory(
         commitMessage, () => {
-        }, (errorMessage: string) => {
-          this.alertsService.addInfoMessage(errorMessage, 5000);
+        // The type of errorMessage is unknown because error messages
+        // can be of type anything. For example, it can be a string
+        // or an object.
+        }, (errorMessage: unknown) => {
+          this.alertsService.addInfoMessage(errorMessage as string, 5000);
```

Comment:
Hmm, rather than doing `as string` can we verify that the return type is actually string? Why is this `unknown` only here and not in other places like this?

Reply:
Done.